pyproject.toml

```diff
@@ -144,6 +144,7 @@ pyprof2calltree = "*"
 pytest = "*"
 pytest-cov = "*"
 pytest-cpp = "*"
+pytest-forked = "*"
```

Comment:
i'm building a new agnos. want to get this in?

Reply:
we probably won't use it for any of the tici tests, but it will be nice to have it in case

Reply:
and pytest-randomly for test randomization?

Reply:
https://github.com/commaai/agnos-builder/commit/0e61fa851075220d5aaae48d91b8cf287ae20848